tests/lit/lit/formats/alive2test.py

```diff
@@ -129,6 +126,8 @@ def execute(self, test, litConfig):
     cmd.append(test)
     if alive_tv_2:
       cmd.append(test.replace('.src.ll', '.tgt.ll'))
+    elif alive_tv_3:
+      cmd = cmd + [test, '-always-verify']
```

Comment:
uhm, doesn't this need to be `[test, test, --]` ?

Reply:
The other test is added by line 126.